src/mixins/object_geometry.mixin.js

```diff
@@ -630,9 +738,12 @@
         scaleY: this.scaleY,
         skewX: this.skewX,
         skewY: this.skewY,
+        width: this.width,
+        height: this.height,
+        strokeWidth: this.strokeWidth
```

Comment:
this addition seems not used.
i also remember this was introduced to work around skewX and skewY difficulties nad is already an hack.
Why do we want to exented it to width/height/strokeWidth?

Reply:
It is used by Group/Layer layout logic

Reply:
i see where this is used. Letting go for now, but i ll try my best to remove the need for this, right after this Pr is merged. I don't think this method should exist in this way.